src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/InputPart.Fields.Edit.cshtml

```diff
@@ -0,0 +1,42 @@
+@using OrchardCore.Forms.ViewModels;
+@model InputPartEditViewModel
+
+<fieldset class="form-group">
+    <label asp-for="Type">@T["Type"]</label>
+    <select asp-for="Type" class="form-control content-preview-select">
+        <option value="text">Text</option>
```

Comment:
Localization

Reply:
Unless we talk technical values, but it doesn't look like it.

Reply:
I intentionally used the technical values as the names for the list options, as I thought it might make it easier to recognize what input HTML would actually be rendered.  Then again, I am using capitalization and replaced hyphens with spaces.